modules/post/windows/manage/reflective_dll_inject.rb

```diff
@@ -6,45 +6,167 @@
 require 'msf/core/post/windows/reflective_dll_injection'
 
 class MetasploitModule < Msf::Post
-  include Msf::Post::Windows::ReflectiveDLLInjection
+  Rank = NormalRanking
```

Comment:
Post modules don't get a ranking, that's just for exploits.

Reply:
Ok I will fix